client/components/RatingsAndReviews/Rating/RatingSummaryStars.jsx

```diff
@@ -2,18 +2,18 @@ import React from 'react';
 import Rating from '@material-ui/lab/Rating';
 import { Grid, Container } from '@material-ui/core';
 
-const RatingSummaryStars = ({ ratings }) => {
+const RatingSummaryStars = ({ averageRating }) => {
   return (
-    <>
+    <props>
     <Container>
       <Grid container direction="row" spacing={1}>
-        <Grid item xs={3}><h1>3.5</h1></Grid>
+        <Grid item xs={3}><h1>{averageRating.toFixed(1)}</h1></Grid>
         <Grid item xs={4}>
-        <Rating name="read-only" readOnly value={3.5} />
+        <Rating name="read-only" readOnly value={averageRating} />
         </Grid>
       </Grid>
     </Container>
-    </>
+    </props>
```

Comment:
- I'm not exactly sure what the <props> tag is and that seems to be giving a warning in the browser for me.

Reply:
Good catch, no clue how <props></props>   got in there😅

Reply:
It's all good :)